File: php/examples/get_document_page_url.php
<?php
require_once 'base_request.php';

// Example of GET /documents/:document_id/page/:page_number/url

$debug = true;
$document_id   = $_ENV['DOCUMENT_ID'];
$page_number = $_ENV['PAGE_NUMBER'];
$request_data = '';
$request_params = [];

send_request('GET', '/documents/' . $document_id . '/page/' . $page_number . '/url', $request_params, $request_data);

File: csharp/examples/GetDocumentPageUrl.cs
using System;

namespace Examples.examples
{
    // Example of GET /documents/:document_id/page/:page_number/url
    
    class GetDocumentPageUrl : BaseRequest
    {
        private string _documentId;
        private string _pageNumber;

        public GetDocumentPageUrl() : base()
        {
            dotenv.net.DotEnv.Load();
            _documentId = Environment.GetEnvironmentVariable("DOCUMENT_ID");
            _pageNumber = Environment.GetEnvironmentVariable("PAGE_NUMBER");
            Console.WriteLine($"GetDocumentPageUrl: document_id={_documentId} page_number={_pageNumber}");
        }

        public override string SendRequest()
        {
            string path = $"documents/{_documentId}/page/{_pageNumber}/url";
            return GetRequest(path);
        }
    }
}
